src/patternfly/components/MultipleFileUpload/multiple-file-upload.scss

```
@@ -12,40 +13,47 @@
     "title"
     "upload"
     "info";
-  --#{$multiple-file-upload}__main--Gap: var(--#{$pf-global}--spacer--md);
-  --#{$multiple-file-upload}__main--PaddingTop: var(--#{$pf-global}--spacer--lg);
-  --#{$multiple-file-upload}__main--PaddingRight: var(--#{$pf-global}--spacer--lg);
-  --#{$multiple-file-upload}__main--PaddingBottom: var(--#{$pf-global}--spacer--xl);
-  --#{$multiple-file-upload}__main--PaddingLeft: var(--#{$pf-global}--spacer--lg);
-  --#{$multiple-file-upload}__main--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
+  --#{$multiple-file-upload}__main--PaddingTop: var(--pf-t--global--spacer--xl);
+  --#{$multiple-file-upload}__main--PaddingRight: var(--pf-t--global--spacer--xl);
+  --#{$multiple-file-upload}__main--PaddingBottom: var(--pf-t--global--spacer--xl);
+  --#{$multiple-file-upload}__main--PaddingLeft: var(--pf-t--global--spacer--xl);
+  --#{$multiple-file-upload}__main--BorderWidth: var(--pf-t--global--border--width--regular);
   --#{$multiple-file-upload}__main--BorderStyle: dashed;
-  --#{$multiple-file-upload}__main--BorderColor: var(--#{$pf-global}--BorderColor--100);
-  --#{$multiple-file-upload}__main--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$multiple-file-upload}__main--BorderColor: var(--pf-t--global--border--color--default);
+  --#{$multiple-file-upload}__main--BorderRadius: var(--pf-t--global--border--radius--medium);
 
   // title
   --#{$multiple-file-upload}__title--Display: grid;
   --#{$multiple-file-upload}__title--GridTemplateColumns: auto;
-  --#{$multiple-file-upload}__title--Gap: var(--#{$pf-global}--spacer--sm);
+  --#{$multiple-file-upload}__title--Gap: var(--pf-t--global--spacer--lg);
+
+  // title text
+  --#{$multiple-file-upload}__title-text--Color: var(--pf-t--global--text--color--regular);
+  --#{$multiple-file-upload}__title-text--FontFamily: var(--pf-t--global--font--family--heading);
+  --#{$multiple-file-upload}__title-text--FontSize: var(--pf-t--global--font--size--lg);
```

I think this should use the small heading token?

```suggestion
  --#{$multiple-file-upload}__title-text--FontSize: var(--pf-t--global--font--size--heading--sm);
```